src/components/form/HdCheckboxIndicator.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div
+    class="indicator"
+    :class="{
+      ['indicator--selected']: checked,
+      ['indicator--disabled']: disabled,
+      ['indicator--invalid']: invalid,
+      ['indicator--indeterminate']: indeterminate,
+    }"
+  >
+    <div class="indicator__box">
+      <HdIcon v-if="statusIcon" :src="statusIcon" class="indicator__icon" />
+    </div>
+  </div>
+</template>
+
+<script>
+// @ts-check
+import { HdIcon } from 'homeday-blocks/main';
+import { check as checkIcon, minus as minusIcon } from 'homeday-assets';
+
+export default {
+  name: 'HdCheckboxIndicator',
+  components: {
+    HdIcon,
+  },
+  props: {
+    checked: {
+      type: Boolean,
+      required: true,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    invalid: {
+      type: Boolean,
+      default: false,
+    },
+    indeterminate: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    statusIcon() {
+      if (this.indeterminate) return minusIcon;
+      if (this.checked) return checkIcon;
+      return null;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import "homeday-blocks/src/styles/mixins.scss";
+
+.indicator {
```

Comment:
I propose that we remove the `scoped` keyword. This could create an issue because `indicator` is kind of generic, but maybe we should make the class more specific? Perhaps `checkbox-indicator`?

Reply:
Thanks for your suggestion man =D here are the changes

9bb0eee